projects/aca-content/src/lib/components/shared-link-view/shared-link-view.module.ts

```diff
@@ -27,28 +27,16 @@ import { NgModule } from '@angular/core';
 import { SharedLinkViewComponent } from './shared-link-view.component';
 import { CommonModule } from '@angular/common';
 import { CoreModule } from '@alfresco/adf-core';
-import { RouterModule, Routes } from '@angular/router';
 import { DirectivesModule } from '../../directives/directives.module';
 import { AppCommonModule } from '../common/common.module';
 import { AppToolbarModule } from '../toolbar/toolbar.module';
 import { AppInfoDrawerModule } from '../info-drawer/info.drawer.module';
 import { CoreExtensionsModule } from '../../extensions/core.extensions.module';
 
-const routes: Routes = [
```

Comment:
why this is removed?

Reply:
it was moved to routing since we are not lazy loading it